lib/modules/usernameHider.js

```diff
@@ -136,6 +136,7 @@ const hideUsername = _.memoize((user = loggedInUser()) => {
 			background-color: inherit;
 			border-radius: inherit;
 			padding: inherit;
+			margin-right: 0.5em;
```

Comment:
Note: `inherit` doesn't work here because we set the font-size of the parent to 0 above